tests/ui/const-generics/min_const_generics/self-ty-in-const-2.stderr

```diff
@@ -1,14 +1,8 @@
-error: generic `Self` types are currently not permitted in anonymous constants
+error: generic parameters may not be used in const operations
```

Comment:
this feels like an unfortuante diagnostic regression. want to pass in something to `check_param_use_if_mcg` to say whether the term is from a Self alias or not